src/main/java/org/broad/igv/ui/util/SnapshotUtilities.java

```diff
@@ -268,7 +268,7 @@ private static void exportScreenshotSVG(Component target, File selectedFile, int
         }
     }
 
-    private static void paintImage(Component target, Graphics2D g, int width, int height, boolean paintOffscreen) {
+    private static void paintImage(Component target, SVGGraphics2D g, int width, int height, boolean paintOffscreen) {
```

Comment:
Why the change to SVGGraphics2D, what specifically is used that is not defined in Graphics2D?

Reply:
You are right! Fixed, verified that the SVG export still works on IGV and pushed it right now.